Userland/Libraries/LibC/time.cpp

```diff
@@ -208,8 +208,11 @@ char* asctime_r(const struct tm* tm, char* buffer)
     constexpr size_t assumed_len = 26;
     size_t filled_size = strftime(buffer, assumed_len, "%a %b %e %T %Y\n", tm);
 
-    // Verify that the buffer was large enough.
-    VERIFY(filled_size != 0);
+    // If the buffer was not large enough, set EOVERFLOW and return null
```

Comment:
```suggestion
    // If the buffer was not large enough, set EOVERFLOW and return null.
```

Reply:
How do I commit your addition without making a new commit? Is there a way to run git commit --amend with your change, or should I make the change and commit it myself?